typescript/radix-sdk/src/utils/types.ts

```diff
@@ -33,6 +33,11 @@ export enum RadixIsmTypes {
   NOOP_ISM = 'NoopIsm',
 }
 
+export enum RadixTokenTypes {
+  COLLATERAL = 'Collateral',
+  SYNTHETIC = 'Synthetic',
+}
```

Comment:
I think we should have this type in the same file as the `RadixWarpRouteReader` and not export it as sdk users might think that this and the `TokenType` are interchangeable when they are not